src/v1.json

```diff
@@ -895,6 +903,21 @@
         }
       }
     },
+    "legacy-oscontainer": {
+      "$id": "#/properties/legacy-oscontainer",
+      "type": "object",
+      "title": "Legacy oscontainer",
+      "required": [
+        "image"
+      ],
+      "properties": {
+        "image": {
+          "$id": "#/properties/legacy-oscontainer/image",
+          "type": "string",
+          "title": "Image"
+        }
+      }
+    },
```

Comment:
based on https://github.com/coreos/coreos-assembler/pull/3111#issuecomment-1274818871 I would not expect this change to be needed.